### PR TITLE
Ep add connected acct

### PR DESCRIPTION
**Pull request by edpaget**
Adding in actions and refactoring into components, to the connected accounts PR. Separating it out so changes are easier to see.

### CODE REVIEW

File: src/applications/personalization/connected-accounts/components/NoConnectedApps.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    event: 'account-navigation',
+    'account-action': 'view-link',
+    'account-section': 'vets-faqs',
+  });
+};
+
+export function NoConnectedApps({ errors, propertyName }) {
+  let title;
+  if (errors.length > 0) {
+    title = <h3>Something went wrong loading your connected Accounts.</h3>;
```
Comment (juliaelman) [inline src/applications/personalization/connected-accounts/components/NoConnectedApps.jsx]: Change to "Sorry — we can’t seem to find any connected accounts."

File: src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx [hunk 2]
```diff
@@ -31,121 +33,62 @@ class ConnectedAcctApp extends React.Component {
...
+      connectedAccountsView = (
+        <ConnectedApps
+          confirmDelete={this.confirmDelete}
+          accounts={this.props.accounts}
+          propertyName={propertyName}
+        />
+      );
+    } else if (this.props.loading) {
+      connectedAccountsView = (
+        <LoadingIndicator message="Loading your account information..." />
```
Comment (juliaelman) [inline src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx]: s account information/connected accounts

**Review by juliasolorzano — CHANGES_REQUESTED** [review]
Looks good. Added a few small comments.

Also, the current route is set to `/connected-accounts/` instead of `/account/connected-accounts/`. I am wondering if it's possible to remove the routing from vagov-content completely at that is how we get this work properly 🤔 I've seen this done a few ways in apps around this repo. Open to ideas here!

**Comment by edpaget** [comment]
This this just all be set now.

**Review by juliasolorzano — APPROVED** [review]
🚀

**Pull request merged**